c3.js

```diff
@@ -5747,6 +5747,7 @@
         var $$ = this, z = $$.config.zoom_enabled ? $$.zoom : function () {};
         $$.main.select('.' + CLASS.zoomRect).call(z).on("dblclick.zoom", null);
         $$.main.selectAll('.' + CLASS.eventRect).call(z).on("dblclick.zoom", null);
+        $$.config.zoom_onzoom.call($$.api, $$.x.orgDomain());
```

Comment:
I believe you want to add this to `/src/zoom.js` not `c3.js` as this is the result of the build.